src/idiomatic/foundations-api-design/predictable-api/naming-conventions/09-owned.md

```diff
@@ -0,0 +1,21 @@
+---
+minutes: 2
+---
+
+# Owned
+
+Suffix for functions and methods about owned values in a context where
+references are common.
```

Comment:
This is quite a weak pattern, and the word is mostly self-explanatory. I question the value of this slide.

Could we move this slide after the "to" slide, and turn it into a question for the audience: what is the difference between `to_owned` and `into_owned`? (no material on the slide apart from the method names) Expect the answer to discuss self by value vs. by ref.